src/utils/extensions/Sealable.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity 0.8.18;
+
+abstract contract Sealable {
```

Comment:
Maybe add some natspec comments about the purpose of `Sealable`.

Reply:
@utx0 - we can move this into another PR also.

Reply:
done. pushed